Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 members = [
     "rubble",
+    "rubble-derive",
```

Comment:
This doesn't do anything yet, so please remove the crate until we have an actual implementation of this (if we go the build script route this should also be called differently, eg. `rubble-codegen`).